plugins/org.eclipse.n4js.transpiler/src/org/eclipse/n4js/transpiler/print/LineColTrackingAppendable.java

```diff
@@ -51,6 +51,9 @@ public Appendable append(char c) throws IOException {
 
 	@Override
 	public Appendable append(CharSequence csq) throws IOException {
+		if (null == csq) {
+			return this;
+		}
```

Comment:
this fail-safe behavior will just make bugs in the transpiler (or other code) be overlooked and more difficult to debug.
Please remove or throw an exception with a helpful message.